code/MemberImportForm.php

```diff
@@ -37,7 +37,7 @@ public function __construct($controller, $name, $fields = null, $actions = null,
             $importer = new MemberCsvBulkLoader();
             $importSpec = $importer->getImportSpec();
 
-            $columns = implode(', ', array_keys($importSpec['fields']));
+            $columns = implode(', ', array_keys($importSpec['fields'] ?? []));
```

Comment:
This is ultimately the same as the `GroupCsvBulkLoader` import spec as mentioned above.
No need for null coalescing here.